tests/lora/test_lora_layers_flux.py

```diff
@@ -156,6 +183,132 @@ def test_with_alpha_in_state_dict(self):
         )
         self.assertFalse(np.allclose(images_lora_with_alpha, images_lora, atol=1e-3, rtol=1e-3))
 
+    def test_with_norm_in_state_dict(self):
+        components, _, denoiser_lora_config = self.get_dummy_components(FlowMatchEulerDiscreteScheduler)
+        pipe = self.pipeline_class(**components)
+        pipe = pipe.to(torch_device)
+        pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_tensor_inputs(torch_device)
```

Comment:
We should be able to use the dummy inputs here and invoke the `pipeline` after loading the lora weights. I don't think we need to have the inputs only for the `transformer` here.